tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -8526,6 +8526,21 @@ def bufferStoreImpl(tmpSgpr, mubuf):
         mubuf2 = MUBUFModifiers(offen=True, offset12=offset+bps/2, glc=glc, slc=slc, isStore=True)
         module.add(BufferStoreB128(src=vgpr(int(srcVgpr +rpv/2), rpv/2), vaddr=addr0, \
                   saddr=addr1, soffset=tmpSgpr, mubuf=mubuf2, comment=comment))
+      elif bps == 64:
+        # split into four dwordx4 loads. Offset the next by +0.5 bps
```

Comment:
0.25bps

Btw, can we use for-loop to make this code section more clear.

Reply:
yes, we can merge the bps=32 and bps=64 into one condition and extend the new condition to bps is a multiple of 32. _(bps >= 32 and bps % 32 = 0)_

Reply:
please see this commit, [07b4d11aa10ffc9fa6d35243679da8d540d817b8](https://github.com/ROCmSoftwarePlatform/hipBLASLt/pull/279/commits/07b4d11aa10ffc9fa6d35243679da8d540d817b8)